### PR TITLE
JAMES-4025 Drop JMAP draft

### PR DESCRIPTION
**Pull request by vttranlina**
https://issues.apache.org/jira/browse/JAMES-4025

### CODE REVIEW

**Comment by vttranlina** [comment]
A lot of Webadmin integration tests use Jmap Draft for testing
We need to rewrite it to jmap rfc

**Comment by vttranlina** [comment]
> A lot of Webadmin integration tests use Jmap Draft for testing
> We need to rewrite it to jmap rfc

DONE IT in https://github.com/apache/james-project/pull/2192

**Review by chibenwa — APPROVED** [review]

File: server/apps/distributed-app/docs/modules/ROOT/pages/benchmark/james-benchmark.adoc [hunk 1]
```diff
@@ -88,7 +88,6 @@ We did provide a lot of simulations in `org.apache.james.gatling.simulation` pat
 Some symbolic simulations we often use:
 
 * IMAP: `org.apache.james.gatling.simulation.imap.PlatformValidationSimulation`
-* JMAP draft: `org.apache.james.gatling.simulation.jmap.draft.PlatformValidationSimulation`
```
[the remark this answers is not in the document]
Comment (vttranlina) [reply server/apps/distributed-app/docs/modules/ROOT/pages/benchmark/james-benchmark.adoc:91]: issue : https://github.com/linagora/james-gatling/issues/175

**Review by Arsnael — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]

**Comment by Arsnael** [comment]
https://ci-builds.apache.org/job/james/job/ApacheJames/job/PR-2187/5/

`resource eml/htmlMail.eml not found.`

=> Likely a file resource that was in draft and got deleted. Should migrate it :)

**Comment by vttranlina** [comment]
green, I will squash

**Comment by vttranlina** [comment]
@Arsnael can you re-run ci?

**Comment by chibenwa** [comment]
(re-runned)

**Pull request merged**